tutorials/quickstart/tutorial.md

```diff
@@ -316,11 +321,11 @@ NOTE: You may also want to take a look at the ServicePulse window, where you sho
 
 In this tutorial, we explored the basics of how a messaging system using NServiceBus works.
 
-We learned that asynchronous messaging failures in one part of a system can be isolated and prevent the entire system failure. That level of resilience and reliability is not easy to achieve with traditional REST-based web services.
+We learned that asynchronous messaging failures in one part of a system can be isolated and prevent complete system failure. This level of resilience and reliability is not easy to achieve with traditional REST-based web services.
 
 We saw how automatic retries provide protection from transient failures like database deadlocks. If we implement a multi-step process as a series of message handlers, then each step will be executed independently and can be automatically retried in case of failures. This means that a stray exception won't abort an entire process, leaving the system in an inconsistent state.
 
-We saw how the tooling in the Particular Service Platform makes running a distributed system much easier. ServicePulse gives us critical insights into the health of a system, and allows us to diagnose and fix systemic failures. We don't have to worry about data loss–once we redeploy our system, we can replay failed messages in batches as if the error had never occurred.
+We saw how the tooling in the Particular Service Platform makes running a distributed system much easier. ServicePulse gives us critical insights into the health of a system, and allows us to diagnose and fix systemic failures. We don't have to worry about data loss once we redeploy our system, we can replay failed messages in batches as if the error had never occurred.
```

Comment:
Doesn't make sense without the em-dash. It's almost a complete sentence deserving of a period, but I don't want "We don't have to worry about data loss." to stand by itself without immediate "backup" either. It's a very subtle thing.

So this suggestion reverts the change.
```suggestion
We saw how the tooling in the Particular Service Platform makes running a distributed system much easier. ServicePulse gives us critical insights into the health of a system, and allows us to diagnose and fix systemic failures. We don't have to worry about data loss—once we redeploy our system, we can replay failed messages in batches as if the error had never occurred.
```